cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         name: libmodules
       - name: mon-endpoint-volume
         mountPath: /etc/rook
-  hostNetwork: false
+  hostNetwork: true
```

Comment:
Could you also add a comment here but above the `hostNetwork` line.

Reply:
Added.